src/app/window-handler.ts

```diff
@@ -231,12 +231,6 @@ export class WindowHandler {
             ...this.windowOpts, ...getBounds(this.config.mainWinPos, DEFAULT_WIDTH, DEFAULT_HEIGHT),
         }) as ICustomBrowserWindow;
 
-        if (isWindowsOS) {
-            // SDA-1720 when Symphony on secondary screen is wider than main screen, the window on secondary is clapmed to the width of main screen
-            // Only happens on windows (BrowserWindow)
-            this.mainWindow.setBounds(getBounds(this.config.mainWinPos, DEFAULT_WIDTH, DEFAULT_HEIGHT) as Electron.Rectangle);
```

Comment:
@johankwarnmarksymphony is this fix not required for `SDA-1720`?

Reply:
Yes, we can not cause white screen. No way of getting out of the white screen situation. They can file a new sda-1720, it is an edge case

Reply:
Got it 👍